src/main/java/com/sapozhnykov/dao/client/impl/ClientDaoImpl.java

```diff
@@ -7,17 +7,20 @@
 import java.util.List;
 
 public class ClientDaoImpl implements ClientDao {
+    private static ClientDao clientDao = new ClientDaoImpl();
+
     private List<Client> clients = new ArrayList<>();
-    private static long tempID = 1;
+    private static long tempID = 3;
 
-    public ClientDaoImpl() {
-        clients.add(new Client(tempID++,"David" , "Cramer", "0503213232", "david@gmail.com"));
-        clients.add(new Client(tempID++,"Greg" , "Popovich", "0974321222", "greg@gmail.com"));
+    private ClientDaoImpl() {
+        clients.add(new Client(1,"David" , "Cramer", "0503213232", "david@gmail.com"));
```

Comment:
It's bed idea to fulfilled data inside contractor

Reply:
Fixed

Reply:
I mean that each time when you create instance you fulfilled data, it's better to  extract this from class at all

Reply:
okay, fixed